app/src/main/java/fr/free/nrw/commons/contributions/UploadCountClient.java

```diff
@@ -14,10 +14,10 @@
 import fr.free.nrw.commons.concurrency.ThreadPoolExecutorService;
 import timber.log.Timber;
 
-public class UploadCountClient {
+class UploadCountClient {
     private ThreadPoolExecutorService threadPoolExecutor;
 
-    public UploadCountClient() {
+    UploadCountClient() {
```

Comment:
Sorry if I missed something, but is there a reason why this class and its constructor was made non-public?